FreeRTOS/Demo/WIN32-MingW/code_coverage_additions.c

```diff
@@ -94,39 +94,39 @@ UBaseType_t uxDummy = 10;
 	passed in place of a required buffer.  Hence if all passes then none of the
 	|= will be against 0, and ulReturned will still be zero at the end of this
 	function. */
-	ulReturned |= ( uint32_t ) xEventGroupCreateStatic( NULL );
+	ulReturned |= ( uint32_t ) ( uintptr_t ) xEventGroupCreateStatic( NULL );

```

Comment:
Oops, sneaking this change in. Suppresses a pointer-to-int conversion warning. Might be better to make ulReturned `uintptr_t`, here. Might revert

Reply:
uintptr_t would be better.  Also single purpose PR's are easier to review.